CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+- **CUMULUS-775**
+  - Fixed `@cumlus/cmrjs.searchConcept` to search and return CMR results.
+  - Changed `@cumulus/api/endpoints/granules.list` to inject a `concept-id` key
```

Comment:
I see a change here to the collections endpoint, but not to the granules endpoint.